topic-operator/src/main/java/io/strimzi/operator/topic/cruisecontrol/CruiseControlClientImpl.java

```diff
@@ -130,7 +130,9 @@ public String topicConfiguration(List<KafkaTopic> kafkaTopics) {
         
         // send request and handle response
         return httpClient.sendAsync(request, HttpResponse.BodyHandlers.ofString()).thenApply(response -> {
-            LOGGER.traceOp("Response: {}", response);
+            if (response.body() != null) {
+                LOGGER.traceOp("Response body: {}", response.body());
```

Comment:
Here as well as below ... what is the reson to log just the body and not the whole response? I would expect that the code etc. would be interesting as well.

Reply:
Sure, we can log both, the body has to be explicitly logged.